MREUnityRuntime/MREUnityRuntimeLib/Messaging/Protocols/Protocol.cs

```diff
@@ -53,16 +50,10 @@ public void Stop()
 			}
 		}
 
-		private void Conn_OnReceive(string json)
+		private void Conn_OnReceive(Message message)
 		{
 			try
 			{
-#if ANDROID_DEBUG
-				App.Logger.LogDebug($"Recv: {json}");
-#endif
```

Comment:
This is very useful logging for the android build and shouldn't just be removed.  Can this be added back in to print out the message now that this has changed to a message object?